build/rollup.config.js

```diff
@@ -120,7 +122,23 @@ export default inputs.map(input => {
             }),
             json({
                 preferConst: false
-            })
+            }),
+            /**
+             * The following hack is necessary because expressions.js wants to export an object
+             * containing a key named 'default', and expressions/engine.js needs to import the whole thing.
+             * By omitting a line from Rollup's generated bundle, we can preserve the full object.
+             */
+            {
+                name: 'expressions_helper',
+                generateBundle(options, bundle) {
+                    if (options.name.indexOf('expressions/engine') !== -1) {
```

Comment:
I figured out in another project that this won't be good on windows. You should change it to `if (options.name.match(/expressions[\/\\]engine/)) {`

Reply:
or `path.join('expressions', 'engine')` 
but I think having to handle windows paths in build will be painful. We should figure out a solution.

Reply:
I do not feel it is painful. We had to add 2 small things and everything works.